### ISSUE TITLE
Disabling screenshot on ios is not working 

### ISSUE THREAD

**Issue by noureddine1** [prongbang/screen_protector#2]
Disabeling screenshot on ios is not working

**Comment by benoitskipr** [comment]
yes, `protectDataLeakageOff` would be interesting for ios as well, so we could only set it on specific screens, and not once for the whole app

**Comment by benoitskipr** [comment]
(or `disableImageScreen` and variants as it would be easier not to save a state)
